### PR TITLE
Add support for container logging

### LINKED ISSUE #40: Add support for container logging
The expected container log path is `PodSandboxConfig.LogDirectory/ContainerConfig.LogPath`

```
    // Path relative to PodSandboxConfig.LogDirectory for container to store
    // the log (STDOUT and STDERR) on the host.
    // E.g.,
    //     PodSandboxConfig.LogDirectory = `/var/log/pods/<podUID>/`
    //     ContainerConfig.LogPath = `containerName_Instance#.log`
    //
    // WARNING: Log management and how kubelet should interface with the
    // container logs are under active discussion in
    // https://issues.k8s.io/24677. There *may* be future change of direction
    // for logging as the discussion carries on.
    optional string log_path = 11;
```

cc/ @gnawux

### PR DESCRIPTION
**Pull request by feiskyer**
Fixes #40

### CODE REVIEW

**Comment by feiskyer** [comment]
cc @resouer Ready for review.

**Comment by resouer** [comment]
I am re-install my devbox VM, have to review and feedback here tomorrow.

File: pkg/hyper/types/types.proto [hunk 1]
```diff
@@ -847,4 +847,4 @@ service PublicAPI {
     // Version gets the version and apiVersion of hyperd
     rpc Version(VersionRequest) returns (VersionResponse) {}
     // TODO: Auth auths a user to the specified docker registry
-}
+}
```
Comment (feiskyer) [inline pkg/hyper/types/types.proto]: fixed.

**Comment by resouer** [comment]
LGTM

**Pull request merged**